cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala

```diff
@@ -110,33 +104,33 @@ class CallTest extends DocumentingTest {
     }
 
     section("Call a procedure with literal arguments", "call-call-a-procedure-with-literal-arguments") {
-      p("""This calls the example procedure `org.neo4j.procedure.example.addNodeToIndex` using literal arguments.
+      p("""This calls the example procedure `dbms.security.createUser` using literal arguments.
```

Comment:
The security procedures are deprecated in 4.0 (and in fact there was even a plan for complete removal, but it seems more likely we'll just deprecate). We should use some other examples.

Reply:
We need an example that works on a normal database. Security procedures will only work on system databases (if they are not removed entirely). I favor having a user-built procedure, although a very small and simple one.

Reply:
I have another idea. What about the `db.stats.*` procedures? They are new, work on normal databases (not system) and useful to normal users for support requests?

Reply:
@craigtaverner Thank you! I will try it out.

Reply:
@craigtaverner As agreed in discussions yesterday, I will merge this to get the bug out, and raise a new card to find a different example.